File: Python3/Secuenciales/ej14.py
# -*- coding: utf-8 -*-

#Dado un número de dos cifras, diseñe un algoritmo 
#que permita obtener el número invertido.

#pedir numero de dos cifras
#almacenar en una variable
#multiplicar por -1
#mostrar resultado

print("..::Obtener el número inverso::..")
numero = input("Introduce un número de dos cifras")
if((len(numero)<2)or(len(numero)>2)):
	print("El número debe tener dos digitos")
else:
	inverso = int(numero)*(-1)
	print("El inverso de "+str(numero)+" es "+str(inverso))

File: Java/Secuenciales/ej14.java
import java.util.Scanner;

/**
* Dado un número de dos cifras, diseñe un algoritmo que permita obtener el número invertido.
* @author Alberto Miguel Martinez Jimenez
*/

//Almacenar número
//dividir entre 10 para obtener la primera cifra
//el modulo será la segunda cifra
//se suman las dos cifras y el resultado es el número invertido
//mostrar resultado

public class ej14 {
	public static void main(String[] args) {
		int numero;
		int unidad;
		int decena;
		int numeroFinal;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("Introduce un número entero de dos cifras para invertirlo> ");
		numero = scanner.nextInt();
		decena = numero/10;
		unidad = numero%10;
		numeroFinal =(unidad*10)+decena;
		
		System.out.println("El número "+numero+" invertido es "+numeroFinal);
	}
	
}